.gitignore

```
@@ -110,7 +110,13 @@ docker/*
 
 # macOS
 .DS_Store
+*/.DS_Store
+.*/*/.DS_Store
 
 # IDE
 .idea
 *.iml
+*.xcworkspace
+*.xcworkspace/*
+*.xcodeproj
+*.xcodeproj/*
```

Consider simplifying the pattern:
```suggestion
*.xcodeproj
```

or closing the PR as "Rejected"